static/styles/zulip.css

```diff
@@ -33,6 +29,8 @@ html {
 }
 
 body {
+    overflow-x: hidden;
+    overflow-y: auto;
```

Comment:
For standards-mode pages like ours, the scrollbar [belongs on `html`](https://developer.mozilla.org/en-US/docs/Web/API/document/scrollingElement), not `body`.